src/mlpack/core/util/to_lower.hpp

```diff
@@ -0,0 +1,36 @@
+/**
+ * @file to_lower.hpp
+ * @author Himanshu Pathak
+ *
+ * Convert a string to lower-case.
```

Comment:
mlpack seems to use "lowercase" without a space or dash.
```suggestion
 * Convert a string to lowercase.
```